packages/openapi-to-graphql/src/preprocessor.ts

```diff
@@ -484,13 +481,10 @@ export function createDataDef(
     )
     const saneInputName = Oas3Tools.capitalize(saneName + 'Input')
 
-    // Add the names to the master list
-    data.usedOTNames.push(saneName)
```

Comment:
We were adding `usedOTNames` here when we could not guarantee creating a dataDef. In other words, we potentially wasted the namespace here.